src/common/routes/home/DsEditLog.js

```diff
@@ -248,7 +248,7 @@ class DsEditLog extends Component {
                             columns={columns}
                             data={[]}
                             options={{
-                                ajaxURL: `${config.apiUrl}/ds/view/editLog/${this.props.match.params.dsName}/${dsView}/${user.user}`,
+                                ajaxURL: `${config.apiUrl}/ds/view/editlog/${this.props.match.params.dsName}/${dsView}/${user.user}`,
```

Comment:
I think this may not be required. Please try without this change and it should work.